stackit/internal/services/resourcemanager/resourcemanager_acc_test.go

```diff
@@ -56,6 +56,7 @@ func resourceConfig(name string, label *string, members string) string {
 					members = [
 						%[7]s
 					]
```

Comment:
The `members` field (initial members now) will still have priority when specified, right? So in this case the `owner_email` field has no effect, right?test

Reply:
We can remove the `members` field from the test when really removing the field, but was just asking

Reply:
yes, that is true. members field still have precedence over owner_email. 
I kind of added owner_email to be prepared for the removal of members field after deprecation is over.